app/src/main/java/tttTDD/Java/GameInput.java

```diff
@@ -0,0 +1,50 @@
+package tttTDD.Java;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GameInput {
+
+    public static int playerSelection(Scanner input, Display display) {
+        int value;
+
+        try {
+            value = input.nextInt();
```

Comment:
I think these input methods would be clearer if their prompts were collocated. You could make a counter-argument that this class is responsible only for the input and not the prompt, but that's not quite true; once an input fails validation we print an alternative prompt in both of these methods.